fineract-provider/src/main/java/org/apache/fineract/accounting/financialactivityaccount/service/FinancialActivityAccountReadPlatformServiceImpl.java

```diff
@@ -101,8 +101,12 @@ public FinancialActivityAccountData mapRow(final ResultSet rs, @SuppressWarnings
             final Integer financialActivityId = JdbcSupport.getInteger(rs, "financialActivityId");
             final String glAccountName = rs.getString("glAccountName");
             final String glCode = rs.getString("glCode");
-
-            final GLAccountData glAccountData = new GLAccountData(glAccountId, glAccountName, glCode);
+            //@formatter:off
```

Comment:
I'm not sure if we should keep these formatting comments in place... I used them only in my example PR to make sure these parts are highlighted and readable. While I still like the formatting here with those comments, I think we should remove them, because others will copy them to circumvent our general formatting rules.